### PR TITLE
doc: add minutes for meeting Jan 25 2022

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2022-01-25.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+
+### nodejs/build
+
+
+* Platform requirements for Node.js 18 [#2815](https://github.com/nodejs/build/issues/2815)
+  * Planned to last until ~ April 2025, so looking at what platforms which still be in support
+    at that time
+  * Main change is that centos 7 will EOL before then so planning to move to RHEL 8
+  * Applying for no cost RHEL subscription from Open Source program.
```
Comment (richardlau) [inline doc/meetings/build-wg-meeting-2022-01-25.md:28]: We could link to https://www.redhat.com/en/blog/extending-no-cost-red-hat-enterprise-linux-open-source-organizations

```suggestion
  * Applying for no cost [RHEL subscription for Open Source Infrastructure](https://www.redhat.com/en/blog/extending-no-cost-red-hat-enterprise-linux-open-source-organizations) program. 
```

File: doc/meetings/build-wg-meeting-2022-01-25.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* Platform requirements for Node.js 18 [#2815](https://github.com/nodejs/build/issues/2815)
+  * Planned to last until ~ April 2025, so looking at what platforms which still be in support
+    at that time
+  * Main change is that centos 7 will EOL before then so planning to move to RHEL 8
+  * Applying for no cost RHEL subscription from Open Source program. 
+  * For most architectures at least one provider we can install versus provide templates, for the
+    others we can likely build our own images.
+
+* Certificate expiry 18 January 2022 [#2811](https://github.com/nodejs/build/issues/2811)
+  * One report on OSX Monterey of not being able to download headers? But seems strange
```
Comment (richardlau) [inline doc/meetings/build-wg-meeting-2022-01-25.md:33]:
```suggestion
  * One report on macOS Monterey of not being able to download headers? But seems strange
```

File: doc/meetings/build-wg-meeting-2022-01-25.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    since have not seem issues on other OSX levels or any other reports.
```
Comment (richardlau) [inline doc/meetings/build-wg-meeting-2022-01-25.md:34]:
```suggestion
    since have not seem issues on other macOS levels or any other reports.
```

**Review by richardlau — APPROVED** [review]
Minor suggestions, otherwise LGTM.

**Review by mmarchini — APPROVED** [review]

**Pull request merged**